openwisp_monitoring/device/tests/test_api.py

```diff
@@ -218,14 +218,16 @@ def test_200_multiple_measurements(self):
         # expected upload wlan1
         self.assertEqual(data['traces'][1][1][-1], 1.5)
 
-    def test_garbage_clients(self):
```

Comment:
Why did you remove this test?

Reply:
Because we've made all the properties required and have equally defined their schemas, meaning there will never be an instance where the system will allowed wireless interface with garbage_client since the schema validation will fail